docs/gallery/general/file.py

```diff
@@ -387,19 +387,63 @@
 nwbfile.add_unit(id=3, spike_times=[1.2, 2.3, 3.3, 4.5],
                  obs_intervals=[[1, 10], [20, 30]], location='CA1', quality=0.90)
 
-####################
-# Now we overwrite the file with all of the data
-
-with NWBHDF5IO('example_file_path.nwb', 'w') as io:
-    io.write(nwbfile)
-
 ####################
 # .. _units_fields_ref:
 #
 # .. note::
 #    The Units table has some predefined optional columns. Please review the documentation for
 #    :py:meth:`~pynwb.file.NWBFile.add_unit` before adding custom columns.
 
+####################
+# Images
+# ------
+#
+# You can store static images within the NWB file as well. These can be images of the subject, the environment, stimuli,
+# or really anything.
+
+from PIL import Image
+import requests
+import numpy as np
+
+from pynwb.image import RGBAImage, RGBImage, GrayscaleImage
+from pynwb.base import Images
+
+image_url = 'https://flif.info/example-images/fish.png'
+img = Image.open(requests.get(image_url, stream=True).raw)
```

Comment:
For the purpose of testing I would recommend to just add (or use) an image from the static files from the docs. Having to download an image creates an unnecessary dependency on an external service and requires web access and I don't think downloading the image is important for the tutorial.